### PR TITLE
Modified notebook SDK_AutoML_Video_Classification.ipynb

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
Modified notebook SDK_AutoML_Video_Classification.ipynb
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,1141 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Vertex SDK -&gt; Vertex AI SDK</p><p> </p><p>Move title into next cell (combine title + links into single cell)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Move Dataset after Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Add uses section.</p><p> </p><p>Steps could be more descriptive. Remove reference to cleanup</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Drop BigQuery pricing -- not used in notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    ! pip3 install {USER_FLAG} --upgrade google-cloud-aiplatform
</pre><p>remove execution output.</p><p> </p><p>Combine installs into single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Use template version.</p><p>Remove execution output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Use template versions for BUCKET and REGION</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Remove ALL execution outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>MY_PROJECT = PROJECT_ID</p><p>MY_STAGING_BUCKET = BUCKET_URI&nbsp;# bucket should be in same region as ucaip</p><p> </p><p>Above seems unnecessary. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>combine two code cells into one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <h1>Managed Video Dataset -&gt; Vertex AI Video Dataset</h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>use PROJECT_ID and BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Remove future tense 'will'</p><p> </p><h2>Create a Dataset on Vertex AI -&gt; Creare a Vertex AI Dataset resource</h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Remove option 2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>use sentence case for title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Config -&gt; Configure.</p><p> </p><p>Explain parameters for method call</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Explain parameters for method call</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <h1>Batch Prediction Job on the Model -&gt; Make a Batch Prediction request</h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>Sentence case</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>accompanying text cell should explain parameters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>need text cell with explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb:1]: <p>need text cell with explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

**Comment by sudarshan-SpringML** [comment]
@andrewferlitsch all requested changes done

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**